Automattic-Tracks-iOS/Extensions/FileManager+Extensions.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+
+extension FileManager {
+
+    var documentsDirectory: URL {
+        let documentsDirectory = NSSearchPathForDirectoriesInDomains(.documentDirectory, .userDomainMask, true).first!
+        return URL(fileURLWithPath: documentsDirectory, isDirectory: true)
+    }
+
+    func contentsOfDirectory(at url: URL) throws -> [URL] {
+        return try FileManager.default.contentsOfDirectory(atPath: url.path).map { url.appendingPathComponent($0) }
```

Comment:
Do we really want to have the `FileManager.default.` bit in these methods? Shouldn't we call the methods on this particular `FileManager` instead or if we really want the `FileManager.default` (which I don't think we do) shouldn't this be a static function?

This comment applies to other instances of `FileManager.default` in this extension.

Reply:
Ah – you're totally right about this – we should use the same instance.

Fixed in de728bd345be5c5615211fb0b03cbb3bbd6a94fb.